website/developer-docs/docs/writing-documentation.md

```diff
@@ -28,48 +31,25 @@ The site is built using npm, below are some useful make commands:
-
-    -   DON'T: "The Applications page will be loaded."
-
-    -   DO: "The Applications page displays."
-
--   Phrasing should never blame the user, and should be subjective:
-
-    -   DON'T: "Never modify the default file."
-
-    -   DO: "We recommend not modifying the default file."
-
--   When referring to UI text or UI components in authentik, use **bold** text.
--   When referring to other objects in authentik code or functionality, use _cursive_ text, and link to the corresponding documentation if possible.
--   When referring to external tools, give an example how to use the tools or explain how the user can use them.
--   Make sure to add the documentation to the sidebar, if adding a new page.
--   Test how the documentation renders using the Netlify Deploy Preview, especially when using Docusaurus-specific features. Or build it locally and test, using the `make website-watch` command.
+## Writing guidelines
 
-If you find any documentation that doesn't match these guidelines, feel free to either open an [Issue](https://github.com/goauthentik/authentik/issues) or a [PR](https://github.com/goauthentik/authentik/pulls) so they can be fixed.
+Please refer to our [Style Guide](./style-guide.mdx) for authentik documentation. Here you will learn important guidelines about not capitalizing authentik, how we format our titles and headers, and much more.
 
-## Integration guidelines
+Whenever possible, use one of our [docs templates](./templates/index.md). This makes it a lot easier for you (no blank page frights!) and keeps the documentation consistent.
 
-These guidelines apply in addition to the ones above.
+Following the guidelines will make getting your PRs merged much easier and faster, with fewer edits needed. We appreciate our community contributors helping us keep the Docs consistent, easy-to-use, and high quality.
 
-See the template in `/website/integrations/_template/service.md`.
+## Documentation for Integrations
 
--   For placeholders, use angle brackets and italicize the text inside the brackets, to indicate that it is a variable (`_<placeholder-name>_`).
+In addition to following the [Style Guide](./style-guide.mdx) please review the following guidelines.
 
-    Make sure to also define if the placeholder is something the user needs to define, is something another system defines, or is randomly generated.
+For new integration documentation, please use the Integrations template in our [Github repo](https://github.com/goauthentik/authentik) at `/website/integrations/_template/service.md`.
 
-    If you're adding configuration snippets to the documentation, and the snippet is in a language that supports comments, other placeholders may be used, for example comments referencing an earlier step.
+-   Make sure to add the service to a fitting category in `/website/sidebarsIntegrations.js`. If you do not do this, the service will not appear in the table of contents to the left.
```

Comment:
I think in this wording we're kinda contradicting ourselves in regards to the "never blame the user" part, so probably should be more "If this is not done"